pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ServerCnx.java

```diff
@@ -1760,6 +1762,20 @@ protected void handleSend(CommandSend send, ByteBuf headersAndPayload) {
             printSendCommandDebug(send, headersAndPayload);
         }
 
+
+        ServiceConfiguration conf = getBrokerService().pulsar().getConfiguration();
+        if (producer.getTopic().isFenced()
+                && ExtensibleLoadManagerImpl.isLoadManagerExtensionEnabled(conf)) {
```

Comment:
Do we need to add a boolean field to cache the result of the check `ExtensibleLoadManagerImpl.isLoadManagerExtensionEnabled(conf)`?